src/tests/alias.rs

```diff
@@ -42,6 +42,7 @@ fn alias() {
         ResolveOptions {
             alias: vec![
                 ("aliasA".into(), vec![AliasValue::from("a")]),
+                ("@adir/*".into(), vec![AliasValue::from("./a/")]),
```

Comment:
Notice here is no `*` after `./a/`, user needs to preprocess it, I am not sure if we need to put that parsing logic in `oxc-resolver`